virtual_rainforest/models/animals/animal_communities.py

```diff
@@ -97,11 +97,16 @@ def die_cohort(self, cohort: AnimalCohort) -> None:
 
         if cohort.is_alive:
             cohort.is_alive = False
-            LOGGER.debug("An animal cohort has died")
+            # LOGGER.debug("An animal cohort has died")
             self.animal_cohorts[cohort.name].remove(cohort)
         elif not cohort.is_alive:
             LOGGER.exception("An animal cohort which is dead cannot die.")
 
+    def die_cohort_community(self) -> None:
+        """This handles die_cohort for all cohorts in a community."""
+        for cohort in chain.from_iterable(self.animal_cohorts.values()):
```

Comment:
Given that you iterate over all animal cohorts in a couple of places, how about adding a helper method/property for it?

E.g.:

```py
@property
def all_animal_cohorts(self) -> Iterable[AnimalCohort]:
    return chain.from_iterable(self.animal_cohorts.values())
```